Source/Maestro.H

```diff
@@ -57,6 +68,13 @@ public:
 
     static amrex::Real getCPUTime();
 
+#ifdef AMREX_USE_CUDA
+    // Minimum CUDA threadblock size.
+    static int numThreadsMin[3];
```

Comment:
Is the AMReX-provided functionality for the minimum number of threads insufficient?

Reply:
I think this is left over from when I was trying to work out how to offload `estdt` - I don't actually use it anywhere so this can be deleted.